### ISSUE TITLE
Specify additional/secondary information for title page

### ISSUE THREAD

**Issue by rbeezer** [rbeezer/mathbook#74]
Two use cases:

1.  An author of a supplement or appendix who is credited here.

2.  "Submitted in partial fulfillment...." - Does this part of a thesis typically go on the title page?

Maybe part of this work should include "Originally published as..." that typically goes footnote-style on the same page as most of this information.

**Comment by rbeezer** [comment]
Another use case - giving credit to both original author and a translator on a translation of an existing work.

**Comment by davidfarmer** [comment]
I looked at a bunch of books and also re-read the appropriate sections of two books on book design.

Not much goes on a title page.  Credit to the translator and the illustrator are common, as is credit to
the editor even when there is an author (e.g., a scholarly volume of a famous old book).  But an
open source book with a huge number of contributors could break the age-old model if you try
to list everyone.

Authors of intruductions/prefaces/appendices can go on the title page.

The "Submitted in partial fulfillment..." has no place on the title page of a book.  But the submitted
thesis is not a book.

The book "Bookmaking" by Marshall Lee writes
  
The title page may contain some or all of the following elements:

a) title

b) author

c) subtitle

d) credits (translator, editor, illustrator, author of introduction and/or foreward and/or preface)

e) publisher's name

f) publisher's trademark

g) date

h) copyright notice

i) quotation

j) illustration

Note however (back to Farmer writing, not quoting Lee) the copyright notice typically goes elsewhere, specifically the back side of the title page, assuming the title page is a right-hand page.

**Comment by davidfarmer** [comment]
I don't think you want full contact information on the title page.  That can go on the copyright page.

**Comment by rbeezer** [comment]
Thanks, David - that is very helpful.

I think I will associate the authors' names with the "titlepage" element, but 
migrate most of the contact info to the copyright page.

And you are certainly right about a thesis - maybe someday there will be a 
"thesis" element.

Rob

**Comment by rbeezer** [comment]
Making a **single** `image` possible might be a good addition.
